Generator/Templates/GObject/signal.sbntxt

```diff
@@ -31,52 +30,48 @@ else
     $signal_generic_args = $class_name
 end
 ~}}
+#region Signal {{$signal_name}}
 
-        #region Signal {{$signal_name}}
-        
-        {{~ if $signal.needs_signal_args ~}}
-        ///<summary>
-        /// Indexer to connect {{ signal_descriptor_name }} with an SignalHandler<{{$signal_generic_args}}>
-        ///</summary>
-        public SignalHandler<{{ $signal_generic_args }}> this[Signal<{{ $signal_generic_args }}> signal]
-        {
-            set => signal.Connect(this, value, true);
-        }
-        
-        /// <summary>
-        /// Signal (Event) Arguments for {{signal_name}}
-        /// </summary>
-        public sealed class {{$signal_type}} : SignalArgs
-        {            
-            #region Properties
-            {{~ 
-            index = 0
-            for $param in $signal.parameters.parameters 
-                index = index + 1
-                $signal | write_summary ($indent + '    ')
-            ~}}
-            public {{ $param | write_type }} {{ $param.name | fix_identifier | make_pascal_case }} => Args[{{index}}].Extract<{{ $param | write_type }}>();
+{{~ if $signal.needs_signal_args ~}}
+///<summary>

```

Comment:
There is no space between the triple slash and the `summary` tag.

Reply:
Done